ansible/roles/os_update_latest/tasks/main.yml

```diff
@@ -7,3 +7,4 @@
     name:  "{{ oul_package_names }}"
     exclude: "{{ oul_exclude }}"
     state: latest
+    when: os_update is True
```

Comment:
The `when` should be scoped in the task, it is currently scoped in the `yum` module.

Short answer: align the `when` with the words `name` and `yum` (same level). Basically, remove 2x spaces.